firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -114,10 +116,13 @@ public void setup() throws Exception {
                 .setApiKey(TEST_API_KEY)
                 .build());
 
+    // this is spy instead of mock to be able to test isTesterSingedIn functionality
+    signInStorage = spy(new SignInStorage(firebaseApp.getApplicationContext()));
```

Comment:
I'm surprised the mock doesn't work, since we are passing in the signInStorage class into the constructor. What happens when you mock this class?